app/lib/config/config.go

```diff
@@ -237,3 +240,13 @@ func (s *ServerToml) String() string {
 	}
 	return buff.String()
 }
+
+// TildeToHome takes a path and replaces an eventual "~" with the home-directory
+func TildeToHome(path string) string {
```

Comment:
Mhhh I don't like doing these things manually but apparently it seems difficult to do otherwise. Look [here](https://travis-ci.org/dedis/cothority/builds/143923343).
Any way, in the link, either the accepted solution or [the one below](http://stackoverflow.com/a/17617721) are simpler.
And why would you want to `ErrFatal` ? It seems to me an usual situation...